microsoft-edge/test-and-automation/devtools-protocol.md

```diff
@@ -5,10 +5,10 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 11/19/2021
+ms.date: 1/30/2023
 ---
 # DevTools Protocol
 
-Use the DevTools Protocol to instrument, inspect, debug, and profile browsers including Microsoft Edge.  The Microsoft Edge DevTools Protocol matches the APIs of the Chrome DevTools Protocol.
+Use the DevTools Protocol to instrument, inspect, debug, and profile browsers, including Microsoft Edge.  By [building Microsoft Edge on the Chromium open-source project](https://github.com/MicrosoftEdge/MSEdge/blob/master/README.md), the Microsoft Edge DevTools Protocol matches the APIs of the Chrome DevTools Protocol.
```

Comment:
```suggestion
Use the DevTools Protocol to instrument, inspect, debug, and profile browsers, including Microsoft Edge.  By building Microsoft Edge on the Chromium open-source project, the Microsoft Edge DevTools Protocol matches the APIs of the Chrome DevTools Protocol.  For information about the Chromium open-source project, see [Microsoft Edge and Chromium Open Source: Our Intent](https://github.com/MicrosoftEdge/MSEdge/blob/master/README.md).
```
Standard link pattern to show the destination text as link text. Corp style says do that if possible.